### ISSUE TITLE
Button Changing Size When Edited

### ISSUE THREAD

**Issue by EnableServices** [mautic/mautic#13906]
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.0.30

### What browsers are you seeing the problem on?

Not relevant

### What happened?

When editing the text of a button the element gains extra padding which cannot be reduced.

### How can we reproduce this issue?

- Go to Emails module
- Select to create a new Email
- Choose either Template or Segment Email
- Select any provided template
- Drag on a button block
- Edit the Text and click save

Expected:
Text displays and padding remain the same

Actual:
Text displays and extra padding is applied which cannot be removed
![button after](https://github.com/mautic/mautic/assets/93786596/d3a48cff-d065-4045-b88a-916c70d0be1b)
![button before](https://github.com/mautic/mautic/assets/93786596/79e4079e-e60b-4c56-8246-1355187633a2)

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
Pretty sure this is a duplicate of https://github.com/mautic/mautic/issues/13782 - closing this so we can continue the discussion there.
